examples/local_orion/diffusion/run_orion.sh

```diff
@@ -0,0 +1,16 @@
+export ORION_DB_ADDRESS='orion_db.pkl'
+export ORION_DB_TYPE='pickleddb'
+
+CONFIG=config_diffusion.yaml
+DATA_DIR=../../data/si_diffusion_small
```

Comment:
This should be `../../../data/si_diffusion_small`

Reply:
I suggest that you define a variable 
> ROOT_DIR=../../../
and use it to define both `DATDA_DIR` and the path to training script.